library/src/main/java/com/mux/player/MuxPlayer.kt

```diff
@@ -138,6 +152,7 @@ class MuxPlayer private constructor(
     private var enableSmartCache: Boolean = false
     private var logger: Logger? = null
     private var customerData: CustomerData = CustomerData()
+    private var didAddMonitoringData: Boolean = false
```

Comment:
question: The value is never read, was this no longer required?

Reply:
It's used here, I think? https://github.com/muxinc/mux-player-android/blob/b95ac515533baf4f3e81a5f69cdc05d5bf6ac853/library/src/main/java/com/mux/player/MuxPlayer.kt#L262

Reply:
I meant this to work the same way that `MuxPlayer.Builder` `enableSmartCache` does.

Reply:
ahh, understood! I must have missed that